src/sql/workbench/contrib/charts/browser/charts.contribution.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Extensions, IConfigurationRegistry, IConfigurationNode } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+import * as nls from 'vs/nls';
+
+const configurationRegistry = Registry.as<IConfigurationRegistry>(Extensions.Configuration);
+
+const chartsConfiguration: IConfigurationNode = {
+	id: 'charts',
+	type: 'object',
+	title: nls.localize('chartsConfigurationTitle', "Charts"),
```

Comment:
I wonder if this should be put under query editor settings along with other stuff like default chart type.

Although now that notebooks renders charts too I wonder if we should just split these into their own section `Query Results Charts` or something...